libs/community/langchain_community/tools/pangea/domain_intel_guard.py

```diff
@@ -97,7 +97,7 @@ def _run(self, input_text: str) -> str:
 
         # Check if the score is higher than the set threshold for any domain
         if any(domain_data.score >= self._threshold for domain_data in intel.result.data.values()):
-            raise PangeaDomainGuardError("Malicious domains found in the provided input.")
+            input_text = "Malicious domain(s) found in the provided input."
 
         # Return unchanged input_text
```

Comment:
nitpick: this comment isn't necessarily try anymore since line 100 is now changing `input_text`